src/media/css/detail/more-info.styl

```diff
@@ -5,10 +5,14 @@
 @import '../lib';
 
 .app-more-metadata {
+    margin: 0 -12px;
```

Comment:
We normally sort our CSS properties alphabetically. So `font-size` should come before `margin`. `padding` is out of place in the `dl` styles also. Sorry I should have mentioned this before.